feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -244,7 +244,10 @@ export class AddToCartComponent implements OnInit, OnDestroy {
   }
 
   onPickupOptionsCompLoaded() {
-    if (this.featureConfigService.isEnabled('a11yPickupOptionsTabs')) {
+    if (
+      this.featureConfigService.isEnabled('a11yPickupOptionsTabs') &&
+      this.pickupOptionCompRef instanceof ComponentRef
```

Comment:
This change is not directly related to the issue, but still its better to have this fix as soon as possible, since there is an error in console for b2b